datasources.yaml

```
@@ -381,4 +381,11 @@ domains:
           data: observations
           next: paging.next
           is_last: paging.isLast
+        
+  logement_social:
+      all:
+        filename: logements_sociaux_2024.ipynb
+        description: recuperation des indicateurs des logements sociaux via notebook
+        type: NotebookExtractor
+        format: json
```

Ce n'est pas un csv ici que tu extraits via le notebook ? "lgt_soc_2024_extracted.csv" ?

```suggestion
        format: csv
```

---

Yes pour le csv.
Par contre ici j'ai repris ce notebook sans m'être concerté avec l'équipe logement sociaux donc je sais pas trop ce qu'il y a dedans.
Je pense que je vais ajouter de la doc mais que je vais retirer cet extract.